barman/output.py

```diff
@@ -709,17 +709,52 @@ def render_show_backup_general(backup_info, output_fun, row):
         :param str row: format string which allows for `key: value` rows to be
             formatted
         """
-        if "backup_name" in backup_info and backup_info["backup_name"] is not None:
-            output_fun(row.format("Backup Name", backup_info["backup_name"]))
+        backup_name = backup_info.get("backup_name")
+        if backup_name:
+            output_fun(row.format("Backup Name", backup_name))
+
         output_fun(row.format("Server Name", backup_info["server_name"]))
-        if backup_info["systemid"]:
-            output_fun(row.format("System Id", backup_info["systemid"]))
+
+        system_id = backup_info.get("systemid")
+        if system_id:
+            output_fun(row.format("System Id", system_id))
+
         output_fun(row.format("Status", backup_info["status"]))
         if backup_info["status"] in BackupInfo.STATUS_COPY_DONE:
             output_fun(row.format("PostgreSQL Version", backup_info["version"]))
             output_fun(row.format("PGDATA directory", backup_info["pgdata"]))
-            output_fun(row.format("Checksums", backup_info["data_checksums"]))
-            output_fun("")
+            cluster_size = backup_info.get("cluster_size")
+            if cluster_size:
+                output_fun(
+                    row.format("Estimated Cluster Size", pretty_size(cluster_size))
+                )
+        output_fun("")
+
+    @staticmethod
+    def render_show_backup_server(backup_info, output_fun, header_row, nested_row):
+        """
+        Render server metadata in plain text form.
+
+        :param dict backup_info: a dictionary containing the backup metadata
+        :param function output_fun: function which accepts a string and sends it to
+            an output writer
+        :param str header_row: format string which allows for single value header
+            rows to be formatted
+        :param str nested_row: format string which allows for `key: value` rows to be
+            formatted
+        """
+
+        data_checksums = backup_info.get("data_checksums")
+        summarize_wal = backup_info.get("summarize_wal")
+        if data_checksums or summarize_wal:
+            output_fun(header_row.format("Server information"))
+            if data_checksums:
+                output_fun(
+                    nested_row.format("Checksums", backup_info["data_checksums"])
+                )
+            if summarize_wal:
+                output_fun(nested_row.format("WAL summarizer", summarize_wal))
+        output_fun("")
```

Comment:
```suggestion
            output_fun("")
```
This needs to be inside the `if` statement so we avoid 2 line-breaks when both checksums and summarize are `None`.